index.html

```diff
@@ -1149,12 +1151,54 @@ <h3>Types</h3>
         </p>
       </section>
 
+
+
+      <section>
+        <h3>Credential Subject</h3>
+
+        <p>
+A <a>verifiable credential</a> contains <a>claims</a> about a <a>subject</a>.
+The <a>subject</a> of the credential MUST be expressed using the
```

Comment:
Same comment as above.
A verifiable credential MUST have a credentialSubject property. The value of the credentialSubject property is the set of claims made about each subject by the issuer..

Reply:
Fixed by your PR.